pkg/kn/commands/flags/sink.go

```diff
@@ -105,3 +105,18 @@ func parseSink(sink string) (string, string) {
 		return parts[0], parts[1]
 	}
 }
+
+// SinkToString prepares a sink for list output
+func SinkToString(sink duckv1.Destination) string {
+	if sink.Ref != nil {
+		if sink.Ref.Kind == "Service" {
+			return fmt.Sprintf("svc:%s", sink.Ref.Name)
+		} else {
+			return fmt.Sprintf("%s:%s", sink.Ref.Kind, sink.Ref.Name)
```

Comment:
can we make the kind lower case ? That's how we use it normally.

Reply:
Done. covert to lower case in the latest version.